docs/constitution/pyth-dao-constitution.md

```diff
@@ -47,6 +51,12 @@ Each PIP must be labeled as Constitutional PIPs or Operational PIPs as described
         - the upgrade of the verification program for each of the blockchains where Pyth data is accessible
         - the setting of data request fees per blockchain, as well as other protocol or network fees
         - the management of PGAS allocation and delegation to validators
+        - the upgrade of the ER program for each of the blockchains where ER is available
+        - the assignment of the relayer function and monitoring of the performance of such assignment
+        - the setting of the ER fee distribution
+        - the upgrade of the Entropy program for each of the blockchains where Entropy is available
+        - the setting of Entropy request fees for each of the blockchains where Entropy is available
+        - the determination of slashing amounts from the amount staked in OIS in compliance with the slashing rules and procedures defined by the DAO in [https://www.notion.so/pyth-network/Rulebook-Pyth-Staking-and-Slashing-for-Data-Integrity-52686ae9ffef4ef999c2febccecc3f5e](TempLink)
```

Comment:
setting the parameters of OIS including:
- delegation fee
- paramaters that determine the capacity of each pool
- reward rate of return